docs/content/users/quickstart.md

```diff
@@ -397,6 +397,37 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
     ddev launch
     ```
 
+=== "Laravel Installer"
+
+    ```bash
+    mkdir my-laravel-site && cd my-laravel-site
+    ddev config --project-type=laravel --docroot=public
+
+    # Temporarily add the Laravel installer
+    echo 'ARG COMPOSER_HOME=/usr/local/composer
+    RUN composer global require laravel/installer
+    RUN ln -s $COMPOSER_HOME/vendor/bin/laravel /usr/local/bin/laravel
+    ' > .ddev/web-build/Dockerfile.laravel
+
+    # Start the project
+    ddev start
+
+    # Select a starter kit of your choice.
+    # The database is temporarily set to SQLite and will be switched to MariaDB
+    ddev exec laravel new temp --database=sqlite
+
+    # Move the installed files
```

Comment:
This needs a better, slightly longer explanation, since people may not understand either that the files are currently in a temp dir or what rsync is.

Reply:
This is a copy-paste from the Symfony quickstart, I'll add the explanation.